src/applications/ivc-champva/10-10D/pages/Sponsor/sponsorInfoConfig.js

```diff
@@ -10,22 +10,16 @@ import {
 import { sponsorWording } from '../../helpers/utilities';
 
 function descriptionText(formData) {
-  // Base: if sponsor
-  let wording =
-    'Please provide your information. We use this information to identify eligibility.';
-
   // Adjust text if is applicant or third party
-  if (formData.certifierRole !== 'sponsor') {
-    const isApp = formData.certifierRole === 'applicant';
-    wording = `Enter the personal information for ${
-      isApp ? 'your' : 'the'
-    } sponsor (the Veteran or service member that ${
-      isApp ? 'you’re' : 'the applicant is'
-    } connected to). We’ll use the sponsor’s information to confirm ${
-      isApp ? 'your' : 'their'
-    } eligibility for CHAMPVA benefits.`;
-  }
-  return wording;
+  const isApp =
+    formData?.certifierRelationship?.relationshipToVeteran?.applicant;
+  return `Enter the personal information for ${
+    isApp ? 'your' : 'the'
+  } sponsor (the Veteran or service member that ${
+    isApp ? 'you’re' : 'the applicant is'
+  } connected to). We’ll use the sponsor’s information to confirm ${
+    isApp ? 'your' : 'their'
+  } eligibility for CHAMPVA benefits.`;
```

Comment:
Simplifying this logic since we no longer have a `certifierRole` property to check.